Tests/test_image.py

```diff
@@ -608,6 +609,13 @@ def act(fp):
 
             assert not fp.closed
 
+    def test_pillow_version(self):
+        with pytest.warns(DeprecationWarning):
+            assert PIL.__version__ == PIL.PILLOW_VERSION
+
+        with pytest.warns(DeprecationWarning):
+            assert int(PIL.PILLOW_VERSION[0]) >= 7
```

Comment:
This'll fail in Pillow 10.x :)

But that's ok, if it's still around in Pillow 10, the failure will be a good reminder to remove it!

Reply:
The line was inspired by https://github.com/pytorch/vision/blob/3c254fb7af5f8af252c24e89949c54a3461ff0be/torchvision/transforms/functional.py#L858 ... so, yeah